test/model-method.js

```diff
@@ -101,3 +101,220 @@ describe('ModelMethod', function() {
+          if (err) return done(err);
+          expect(list).to.have.length(1);
+          expect(list[0]).to.have.property('name', 'testMethod');
+          expect(list[0]).to.not.have.property('isStatic');
+
+          var cfg = new ConfigFile({ path: 'server/models/user.json' });
+          cfg.load(function(err) {
+            if (err) return done(err);
+            var methods = cfg.data.methods;
+            expect(methods).to.be.an('object');
+            expect(methods).to.have.property('testMethod');
+            expect(methods).to.not.have.property('prototype.testMethod');
+            expect(methods.testMethod).to.not.have.property('isStatic');
+            done();
+          });
+        });
+      });
+  });
+  
+  it('add prototype method without isStatic flag to model definition', function(done) {
+    ModelMethod.create(
+      {
+        modelId: userModel.id,
+        name: 'prototype.testMethod',
+        isStatic: false
+      },
+      function(err) {
+        if (err) return done(err);
+        userModel.methods(function(err, list) {
+          if (err) return done(err);
+          expect(list).to.have.length(1);
+          expect(list[0]).to.have.property('name', 'prototype.testMethod');
+          expect(list[0]).to.not.have.property('isStatic');
+
+          var cfg = new ConfigFile({ path: 'server/models/user.json' });
+          cfg.load(function(err) {
+            if (err) return done(err);
+            var methods = cfg.data.methods;
+            expect(methods).to.be.an('object');
+            expect(methods).to.have.property('prototype.testMethod');
+            expect(methods).to.not.have.property('testMethod');
+            expect(methods['prototype.testMethod']).to.not.have.property('isStatic');
+            done();
+          });
+        });
+      });
+  });
+  
+  it('add `prototype.` to method name if isStatic flag is false', function(done) {
+    ModelMethod.create(
+      {
+        modelId: userModel.id,
+        name: 'testMethod',
+        isStatic: false
+      },
+      function(err) {
+        if (err) return done(err);
+        userModel.methods(function(err, list) {
+          if (err) return done(err);
+          expect(list).to.have.length(1);
+          expect(list[0]).to.have.property('name', 'prototype.testMethod');
+          expect(list[0]).to.not.have.property('isStatic');
+
+          var cfg = new ConfigFile({ path: 'server/models/user.json' });
+          cfg.load(function(err) {
+            if (err) return done(err);
+            var methods = cfg.data.methods;
+            expect(methods).to.be.an('object');
+            expect(methods).to.have.property('prototype.testMethod');
+            expect(methods).to.not.have.property('testMethod');
+            expect(methods['prototype.testMethod']).to.not.have.property('isStatic');
+            done();
+          });
+        });
+      });
+  });
+});
+
+function givenWorkspaceWithCustomDependencies(templateName, deps, cb) {
```

Comment:
I think this is generally useful, please move it to `test/support.js`. Consider moving the other two `givenLB*` helpers too.